### PR TITLE
AlmaLinux9 image as default

### PR DESCRIPTION
**Pull request by PMax5**

### CODE REVIEW

File: SwanSpawner/swanspawner/swanspawner.py [hunk 1]
```diff
@@ -86,6 +88,7 @@ def options_from_form(self, formdata):
             options[self.condor_pool]           = formdata[self.condor_pool][0]
             options[self.user_n_cores]          = int(formdata[self.user_n_cores][0])
             options[self.user_memory]           = formdata[self.user_memory][0] + 'G'
+            options[self.use_jupyterlab_field]  = formdata.get(self.use_jupyterlab_field, ['unchecked'])[0]
```
Comment (etejedor) [inline SwanSpawner/swanspawner/swanspawner.py:91]: Why is ['unchecked'] a list?

File: SwanSpawner/swanspawner/templates/options_form_template.html [hunk 2]
```diff
@@ -201,9 +205,24 @@
     <span class='nb' id="optionsHeader">Specify the configuration parameters for the SWAN container that will be created for you.</span>
     </label>
     <label for="alma9">
-    <span class='news' id="alma9">Select <b>AlmaLinux 9</b> as Platform to try out our new experimental Alma9 image with <b>JupyterLab</b> as default interface! More information <a target="_blank" href="https://swan.docs.cern.ch/alma9/">here</a>.</span>
+    <span class='news' id="alma9">Try out our new experimental interface based on <b>JupyterLab</b>!</span>
     </label>
     <br><br>
+    <label> User Interface <a href="#" onclick="toggle_visibility('userInterfaceDetails');"><span class='nbs'>more...</span></a></label>
+    <div style="display:none;" id="userInterfaceDetails">
+        <span class='nb'>The JupyterLab interface offers a more complete development environment for notebooks. Please let us know about any issues you encounter or any feedback you have.</span>
```
Comment (etejedor) [inline SwanSpawner/swanspawner/templates/options_form_template.html:213]:
```suggestion
        <span class='nb'>JupyterLab is the latest web-based interactive development environment for notebooks, code, and data. More information <a target="_blank" href="https://jupyterlab.readthedocs.io/en/stable/user/interface.html">here</a></span>
```

File: SwanSpawner/swanspawner/templates/options_form_template.html [hunk 3]
```diff
@@ -201,9 +205,24 @@
     <span class='nb' id="optionsHeader">Specify the configuration parameters for the SWAN container that will be created for you.</span>
     </label>
     <label for="alma9">
-    <span class='news' id="alma9">Select <b>AlmaLinux 9</b> as Platform to try out our new experimental Alma9 image with <b>JupyterLab</b> as default interface! More information <a target="_blank" href="https://swan.docs.cern.ch/alma9/">here</a>.</span>
+    <span class='news' id="alma9">Try out our new experimental interface based on <b>JupyterLab</b>!</span>
```
Comment (etejedor) [inline SwanSpawner/swanspawner/templates/options_form_template.html:208]:
```suggestion
    <span class='news' id="alma9">Try out our new experimental interface based on <b>JupyterLab</b> and let us know your feedback!</span>
```

File: SwanSpawner/swanspawner/swanspawner.py (hunk 1 above)
Comment (etejedor) [reply SwanSpawner/swanspawner/swanspawner.py:91]:
```suggestion
            options[self.use_jupyterlab_field]  = formdata.get(self.use_jupyterlab_field, 'unchecked')[0]
```

File: SwanSpawner/swanspawner/templates/options_form_template.html [hunk 4]
```diff
@@ -171,25 +171,44 @@
...
+            jupyterLabOption.setAttribute('disabled', '');
+        }
+    }
+
+    function adjust_platforms(jupyterLabOption) {
+        var platformOptions = document.getElementById('platformOptions');
+        var jupyterLabOption = document.getElementById("use-jupyterlab");
+
+        if (jupyterLabOption.checked) {
+            platformOptions.remove(1);
```
Comment (etejedor) [inline SwanSpawner/swanspawner/templates/options_form_template.html:205]: Even better (the code might be cleaner if we do that), when the user selects JupyterLab, could we force the selection of Alma9 and grey the platform menu?

Comment (PMax5) [reply SwanSpawner/swanspawner/templates/options_form_template.html]: I changed it to be like the others, i.e., to grey the platform selection in the menu when users select JupyterLab.

**Review by etejedor — APPROVED** [review]

**Pull request merged**